components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java

```diff
@@ -69,7 +69,7 @@ public int getExecutionOrderId() {
         if (orderId != IdentityCoreConstants.EVENT_LISTENER_ORDER_ID) {
             return orderId;
         }
-        return 100;
+        return 97;
```

Comment:
why did you change the default value?

Reply:
Java 11 detects XML listener file and return the value 97. But in java 8 that value will not detect and return null value for order id.

Reply:
Or else I can change the identity.xml file in the test resources.

Reply:
Updating Mockito version can fix the issue. Changed back the file to original version.